.gitignore

```
@@ -1,2 +1,4 @@
 /node_modules
-/.nyc_output
\ No newline at end of file
+/.nyc_output
+.env
+examples/example-project/node_modules/
```

You shouldn't need to list node_modules twice. Consider switching this file out with https://github.com/github/gitignore/blob/main/Node.gitignore  You should also add `.DS_Store` and remove the one you committed. 
```suggestion
.DS_Store
```